Gemfile.lock

```diff
@@ -1,3 +1,10 @@
+GIT
+  remote: https://github.com/ipaddress-gem/ipaddress
+  revision: a172928b3a29c9286800bf8e4d9a9f13f0cf738a
+  ref: a172928
+  specs:
+    ipaddress (0.8.3)
+
```

Comment:
A gentle reminder to remove this reference before merging.

Reply:
This is intentionally pinned to a Github branch as the source has been updated to include a feature we require here (`.link_local`) but the gem was never released since the update. Only the Gemfile allows pointing to a specific branch/commit like this which is why it sits here and not in the gemspec.

I'll add a comment to clear this up

Reply:
I believe this is merged to master which is released as v0.8.3.

To investigate this I looked at the specific commit that you were referring to
https://github.com/ipaddress-gem/ipaddress/commit/a172928b3a29c9286800bf8e4d9a9f13f0cf738a

Then I looked for these exact changes in the current version. They are here
ipv6: https://github.com/ipaddress-gem/ipaddress/blob/master/lib/ipaddress/ipv6.rb#L426-L453,
ipv4: https://github.com/ipaddress-gem/ipaddress/blob/master/lib/ipaddress/ipv4.rb#L653-L666

Could I overlooked something in this investigation?

Reply:
After discussing in person, I now understand what Farhan meant.

Version 0.8.3 was publish 2 years ago, but this change with `link_local?` is recent and has not been merged. 

[This link](https://github.com/ipaddress-gem/ipaddress/compare/v0.8.3...a172928b3a29c9286800bf8e4d9a9f13f0cf738a) will indicate the time line and real difference.